x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -213,7 +213,8 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
           >
             <EuiBasicTable
               noItemsMessage={
-                status === FETCH_STATUS.LOADING
+                status === FETCH_STATUS.LOADING ||
+                status === FETCH_STATUS.NOT_INITIATED
```

Comment:
Are there scenarios where this will mean the loading spinner will show indefinitely? e.g. by selecting a time range with no data?

Reply:
Hmm.. That's a good point. Still, that feels more like an edge case than the existing behaviour